lib/omnibus/packagers/rpm.rb

```diff
@@ -552,6 +610,29 @@ def with_rpm_signing(&block)
       remove_directory(directory)
     end
 
+    #
+    # Render a file with gpg key passphrase with secure permissions, call the given
+    # block with the path to the file, and ensure deletion of the file from
+    # disk since it contains sensitive information.
+    #
+    # @param [Proc] block
+    #   the block to call
+    #
+    # @return [String]
+    #
+    def with_rpm_passphrase(&block)
+      directory = Dir.mktmpdir
+      passphrase_file = "#{directory}/passphrase"
+      File.open(passphrase_file, 'w', 0700) do |file|
```

Comment:
nit: I think we don't need to mark as executable, right?
```suggestion
      File.open(passphrase_file, 'w', 0600) do |file|
```